spec/features/application/web_server_spec.rb

```diff
@@ -13,11 +13,8 @@
   end
 
   scenario 'visiting an asset with gzip' do
-    asset_file_path = Dir.glob(Rails.root.join('public/assets/*.gz')).first
+    asset_file_path = Dir.glob(Rails.root.join('public/assets/application*.css')).first
```

Comment:
Line is too long. [87/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.